.vscode/launch.json

```diff
@@ -7,23 +7,32 @@
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
+            "args": [
+                "--extensionDevelopmentPath=${workspaceRoot}"
+            ],
             "stopOnEntry": false,
             "sourceMaps": true,
-            "outFiles": [ "${workspaceRoot}/dist/**/*.js" ],
-            "preLaunchTask": "npm: webpack"
+            "outFiles": [
+                "${workspaceRoot}/dist/**/*.js"
+            ],
+            "preLaunchTask": "npm: webpack",
+            "env": {
+                "DEBUG_MODE": "true"
+            }
         },
         {
-			"name": "Run Extension Tests",
-			"type": "extensionHost",
-			"request": "launch",
-			"runtimeExecutable": "${execPath}",
-			"args": [
-				"--extensionDevelopmentPath=${workspaceFolder}",
-				"--extensionTestsPath=${workspaceFolder}/out/test/suite/index"
+            "name": "Run Extension Tests",
+            "type": "extensionHost",
+            "request": "launch",
+            "runtimeExecutable": "${execPath}",
+            "args": [
+                "--extensionDevelopmentPath=${workspaceFolder}",
+                "--extensionTestsPath=${workspaceFolder}/out/test/suite/index"
+            ],
+            "outFiles": [
+                "${workspaceFolder}/out/test/**/*.js"
             ],
-			"outFiles": ["${workspaceFolder}/out/test/**/*.js"],
-			"preLaunchTask": "npm: test-compile"
+            "preLaunchTask": "npm: test-compile"
```

Comment:
Same general comment as with the other ones: please try to keep diffs tidy :) I think the only thing changed here is the addition of an env variable, but figuring that out from this diff is involved and I can't be sure. Would be much easier if the diff had only the line changed that actually changed. I think in general we should try to put reformatting stuff into separate PRs.

Also, we don't need the `DEBUG_MODE` in this PR at all, right? It seems like an unrelated change?

Reply:
yeah, ... the thing is now #1262 _automatically_ formats these things on save (I just forgot to format those JSON files).
I should have commented the main diff here at least, and yeah, I will try to keep diffs tidy.

As for `DEBUG_MODE`, it's what I would like to introduce in this PR. Will explain in another response.

Reply:
Ah, right!